packages/manager/src/components/PaginationControls/PaginationControls.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import { PaginationControls } from './PaginationControls';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof PaginationControls>;
+
+export const PaginationControl: Story = {
+  args: {
+    count: 250,
+    page: 1,
+    pageSize: 25,
+  },
+  render: (args) => {
+    const PaginationControlsWrapper = () => {
+      const [thisPage, setThisPage] = React.useState(args.page);
+
+      const handlePageChange = (pgNum: number) => {
+        setThisPage(pgNum);
+      };
+
+      return (
+        <PaginationControls
+          {...args}
+          onClickHandler={handlePageChange}
+          page={thisPage}
+        />
+      );
+    };
+
+    return <PaginationControlsWrapper />;
+  },
```

Comment:
This is fine, but it's super unfortunate we lose the "Show Code" feature when we make these wrappers. 

I'd prefer

```suggestion
  render: (args) => {
    // eslint-disable-next-line react-hooks/rules-of-hooks
    const [, setArgs] = useArgs();

    return (
      <PaginationControls
        {...args}
        onClickHandler={(page) => setArgs({ page })}
      />
    );
  },
```

but it does have the downside of the `rules-of-hooks` warning.